src/Services/QueuedJobService.php

```diff
@@ -664,7 +664,7 @@ public function runJob($jobId)
                 // have we stalled at all?
                 $stallCount = 0;
 
-                if ($job->SubsiteID && class_exists(Subsite::class)) {
+                if (class_exists(Subsite::class) && isset($job->SubsiteID)) {
```

Comment:
Maybe `!empty($job->SubsiteID)`? I'm not sure if `$job->SubsiteID === 0` was a valid path before but now that would return `true` instead of `false` previously.